### PR TITLE
Custom @ORM\Column annotation

### PR DESCRIPTION
**Pull request by keyermann**
Hello,

Here is a proposition for those who need to customise some ORM\Column annotation to get database types, type options, or to add comments.

### CODE REVIEW

File: src/TypesGenerator.php [hunk 1]
```diff
@@ -755,8 +759,7 @@ private function generateClassUses($annotationGenerators, $classes, $className)
     {
         $uses = $classes[$className]['uses'];
 
-        if (
-            isset($classes[$className]['interfaceNamespace'])
+        if (isset($classes[$className]['interfaceNamespace'])
```
Comment (dunglas) [inline src/TypesGenerator.php]: Why this change?

**Comment by dunglas** [comment]
Can you add a test for this new feature?

**Comment by keyermann** [comment]
If you prefer to have individual options (like nullable, unique), I can do it to.

**Comment by keyermann** [comment]
Yeah I will add test ;)

**Comment by dunglas** [comment]
I like it this way, it will avoid to update the generator every time Doctrine add a new option.

File: src/TypesGenerator.php (hunk 1 above)
Comment (keyermann) [inline src/TypesGenerator.php]: Oh ? I think it's my IDE ! I will force it back. Sorry

Comment (keyermann) [inline src/TypesGenerator.php]: Ok in fact I don't really understand. My code don't have it. 
https://github.com/keyermann/schema-generator/blob/master/src/TypesGenerator.php#L762
Is it possible Scrutinizer do it ?

**Comment by keyermann** [comment]
I put 2 test in your address-book.yml test config : 
```yaml
    [...]
    birthDate: { range: DateTime, ormColumn: 'type="datetimetz", nullable=true, options={"comment" = "Birthdate with timezone."}' }
    [...]
    adminCode: {range: Text, ormColumn: 'type="string", length=3, unique=true, nullable=false, options={"comment" = "A code for central administration."}' }
```

PS : For your information, Travis fail with PHP5.4 with composer install : 

```bash
Your requirements could not be resolved to an installable set of packages.[39;49m

  Problem 1
    - Installation request for symfony/config v3.0.0 -> satisfiable by symfony/config[v3.0.0].
    - symfony/config v3.0.0 requires php >=5.5.9 -> your PHP version (5.4.37) or value of "config.platform.php" in composer.json does not satisfy that requirement.
  Problem 2
    - Installation request for symfony/console v3.0.0 -> satisfiable by symfony/console[v3.0.0].
    - symfony/console v3.0.0 requires php >=5.5.9 -> your PHP version (5.4.37) or value of "config.platform.php" in composer.json does not satisfy that requirement.
  Problem 3
    - Installation request for symfony/event-dispatcher v3.0.0 -> satisfiable by symfony/event-dispatcher[v3.0.0].
    - symfony/event-dispatcher v3.0.0 requires php >=5.5.9 -> your PHP version (5.4.37) or value of "config.platform.php" in composer.json does not satisfy that requirement.
  Problem 4
    - Installation request for symfony/filesystem v3.0.0 -> satisfiable by symfony/filesystem[v3.0.0].
    - symfony/filesystem v3.0.0 requires php >=5.5.9 -> your PHP version (5.4.37) or value of "config.platform.php" in composer.json does not satisfy that requirement.
  Problem 5
    - Installation request for symfony/finder v3.0.0 -> satisfiable by symfony/finder[v3.0.0].
    - symfony/finder v3.0.0 requires php >=5.5.9 -> your PHP version (5.4.37) or value of "config.platform.php" in composer.json does not satisfy that requirement.
  Problem 6
    - Installation request for symfony/process v3.0.0 -> satisfiable by symfony/process[v3.0.0].
    - symfony/process v3.0.0 requires php >=5.5.9 -> your PHP version (5.4.37) or value of "config.platform.php" in composer.json does not satisfy that requirement.
  Problem 7
    - Installation request for symfony/stopwatch v3.0.0 -> satisfiable by symfony/stopwatch[v3.0.0].
    - symfony/stopwatch v3.0.0 requires php >=5.5.9 -> your PHP version (5.4.37) or value of "config.platform.php" in composer.json does not satisfy that requirement.
  Problem 8
    - Installation request for symfony/yaml v3.0.0 -> satisfiable by symfony/yaml[v3.0.0].
    - symfony/yaml v3.0.0 requires php >=5.5.9 -> your PHP version (5.4.37) or value of "config.platform.php" in composer.json does not satisfy that requirement.
  Problem 9
    - Installation request for symfony/translation v3.0.0 -> satisfiable by symfony/translation[v3.0.0].
    - symfony/translation v3.0.0 requires php >=5.5.9 -> your PHP version (5.4.37) or value of "config.platform.php" in composer.json does not satisfy that requirement.
  Problem 10
    - symfony/console v3.0.0 requires php >=5.5.9 -> your PHP version (5.4.37) or value of "config.platform.php" in composer.json does not satisfy that requirement.
    - doctrine/orm v2.5.2 requires symfony/console ~2.5|~3.0 -> satisfiable by symfony/console[v3.0.0].
    - Installation request for doctrine/orm v2.5.2 -> satisfiable by doctrine/orm[v2.5.2].

```

**Comment by dunglas** [comment]
Thank you @keyermann
